contracts/Tokens/VTokens/VToken.sol

```diff
@@ -1121,36 +1133,66 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
             revert("math error");
         }
 
-        BorrowLocalVars memory vars;
+        uint256 totalBorrowsNew;
+        uint256 accountBorrowsNew;
+        if (InterestRateMode(interestRateMode) == InterestRateMode.STABLE) {
```

Comment:
We should use SafeMath in the operations in this block

Reply:
Fixed in [Commit](https://github.com/VenusProtocol/venus-protocol/pull/214/commits/52ba11dd8708d01c72e6d7f2a0a6d3c793303228)